imu_bno055/src/bno055_i2c_activity.cpp

```diff
@@ -158,29 +182,29 @@ bool BNO055I2CActivity::spinOnce() {
         return false;
     }
 
-    sensor_msgs::Imu msg_raw;
+    sensor_msgs::msg::Imu msg_raw;
     msg_raw.header.stamp = time;
     msg_raw.header.frame_id = param_frame_id;
-    msg_raw.header.seq = seq;
+//    msg_raw.header.seq = seq;
```

Comment:
why commented out instead of removed?

Reply:
Yeah, just kill off all of the `seq` stuff; we have no plans to bring it back in ROS 2.